include/bluetooth/mesh/gen_prop_cli.h

```diff
@@ -181,6 +185,10 @@ int bt_mesh_prop_cli_prop_get(struct bt_mesh_prop_cli *cli,
  * not configured.
  * @retval -EAGAIN The device has not been provisioned.
  * @retval -ETIMEDOUT The request timed out without a response.
+ *
+ * The User Property may only be set if the server enabled user write access to
+ * it. If this is not the case, the server will only respond with the set user
+ * access mode for the given property.
```

Comment:
Looks like this was pasted one time too many

Reply:
Oops, fixed!